tools/mo/openvino/tools/mo/moc_frontend/pipeline.py

```diff
@@ -54,6 +57,13 @@ def add_names_to_tensors(model: InputModel, places: List[Place]):
                 log.warn('Could not add an additional name to a tensor pointed to by \'{}\'. Details: {}'.format(
                     new_input['input_name'], str(e)))
 
+    if environ.get('MO_ENABLED_TRANSFORMS') == 'ANALYSIS_JSON_PRINT':
```

Comment:
Also, I think we need some separate routine to register such passes through extensions or so in the future.
@slyalin, please comment on this if you have any thoughts.

Reply:
agree, but it seems to be separate story, should I create a ticket for it?